view/src/App.js

```diff
@@ -0,0 +1,20 @@
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import login from './pages/login';
+import signup from './pages/signup';
+import home from './pages/home';
+
+import React from 'react';
+function App() {
+  return (
+    <Router>
+      <div>
+        <Switch>
+            <Route exact path="/login" component={login}/>
+            <Route exact path="/signup" component={signup}/>
+            <Route exact path="/" component={home}/>
+        </Switch>
+      </div>
+    </Router>
```

Comment:
I think that you can delete `<div>`.  According to the [documentation](https://reactrouter.com/web/api/Switch), `<div>` is not needed if `<Switch>` is used.